cli/package.json

```diff
@@ -43,8 +43,8 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@binaris/oclif-plugin-not-found": "^1.2.4",
-    "@binaris/spice-node-client": "2.0.0",
-    "@binaris/utils-subprocess": "0.0.1",
+    "@binaris/spice-node-client": "~2.2.0",
+    "@binaris/utils-subprocess": "^0.0.1",
```

Comment:
I think this needs to be a fixed version since it points to code in this repo.

Reply:
Done